tests/test_common.py

```diff
@@ -20,12 +20,12 @@ def test_convert_array(self):
         npt.assert_equal(z.dtype, np.float32)
         # now make contiguous
         npt.assert_equal(y.flags.contiguous, False)
-        z = common.convert_array(y, 2, contiguous=True)
+        z = common.convert_array(y, 2)
         npt.assert_equal(z.flags.contiguous, True)
         # test the dim_message
```

Comment:
There isn't a `dim_message` anymore - please remove this portion of the test.

Reply:
We should explicitly test two additional cases: passing in a list of lists, and converting array data of type `np.float64`.